test/resources/PSR with Login/Overview - Invalid date range.bru

```diff
@@ -0,0 +1,26 @@
+meta {
+  name: Overview - Invalid date range
```

Comment:
I'm getting the following response on login:
```
{
  "gatewayToken": "unknown-token"
}
```
It still works, but is the above expected?